interactive_templates/schema/v2.py

```diff
@@ -20,7 +29,7 @@ class Analysis:
     filter_population: str
     repo: str
     time_scale: str
-    time_value: int = field(converter=int)
+    time_value: Optional[int] = field(converter=int_or_none)
```

Comment:
Personal preference, but I think it might be easier to parse if `time_ever: bool = field(converter=bool, default=False)` was grouped with the `time_scale` and `time_value`.

Reply:
Have done, but be aware that order matters in attrs, as you can'd have attributes with no defaults after ones with defaults.  As it happens, we can make this work in this case.